types/mui-datatables/index.d.ts

```diff
@@ -47,7 +48,7 @@ export interface MUIDataTableState {
     data: any[];
     displayData: Array<{ dataIndex: number; data: any[] }>;
     expandedRows: MUIDataTableStateRows;
-    filterData: any[];
+    filterData: string[][];
```

Comment:
https://github.com/gregnb/mui-datatables/blob/02980525e1cb72cc895fa73ad3504567855fab3c/test/MUIDataTableFilter.test.js#L32

Reply:
although `filterData` is initially an empty array, it is populated as a 2D array of strings on [`buildColumns()`](https://github.com/gregnb/mui-datatables/blob/d50ff23334c28035a244233b844501f89afddf7e/src/MUIDataTable.js#L583)